### PR TITLE
Add `molecule status` functional test

### PR DESCRIPTION
**Pull request by retr0h**
Added a `status` functional test for the openstack driver.  This
test caught a bug, where the `inventory_file` was attempting to be
read, when it did not yet exist.

### CODE REVIEW

**Comment by retr0h** [comment]
```
  syntax: commands succeeded
  py27-ansible19-unit: commands succeeded
  py27-ansible19-functional: commands succeeded
  py27-ansible20-unit: commands succeeded
  py27-ansible20-functional: commands succeeded
  py27-ansible21-unit: commands succeeded
  py27-ansible21-functional: commands succeeded
  docs: commands succeeded
  congratulations :)
```

**Comment by alop** [comment]
What happens if there is no inventory file?

**Comment by retr0h** [comment]
> What happens if there is no inventory file?

Reports a proper status.  See the functional tests.

`not_created`

File: tests/functional/openstack/test_command_status.bash [hunk 1]
```diff
@@ -0,0 +1,27 @@
+#!/usr/bin/env bash
```
Comment (alop) [inline tests/functional/openstack/test_command_status.bash]: Personally, I don't like this for bash, but I know why you do it

**Comment by alop** [comment]
Looks fine:+1:

**Pull request merged**